src/loadsave.jl

```diff
@@ -164,7 +164,7 @@ end
 function checkpath_load(file)
     file === nothing && return nothing   # likely stream io
     isa(file, IO) && return nothing
-    !isfile(file) && throw(ArgumentError("No file exists at given path: $file"))
+    startswith(file, "http://") || startswith(file, "https://") || !isfile(file) && throw(ArgumentError("No file exists at given path: $file"))
```

Comment:
The logic seems a little bit hard to understand. If I understand it correctly, is the following an equivalent version?

```suggestion
    startswith(file, "http://") || startswith(file, "https://") || isfile(file) || throw(ArgumentError("No file exists at given path: $file"))
```